app/views/update_mailer/update_identity.html.erb

```diff
@@ -0,0 +1,17 @@
+<p style="  font: 400 1.3em 'Raleway', sans-serif;color: #555;"><h1>
+    Hello <%= @user.name %>,
+ </h1>
+</p>
+<p style="  font: 400 1.1em 'Raleway', sans-serif;color: #555;">
+  MakerRepo website has been updated and you need to update your information.
+  please take take some time and fill the form below.
+</p>
+
+<!--
+should make a form here

```

Comment:
Adding a secure form to an email like this will be a lot of work. It's probably easier to link them to makerepo to update it themselves